sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -4,7 +4,7 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   include(FetchContent)
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  set(UNIFIED_RUNTIME_TAG v0.7)
+  set(UNIFIED_RUNTIME_TAG 47b63869defab0686c58197271050d77ecf3287a)
```

I've created [Prerelease v0.7.1](https://github.com/oneapi-src/unified-runtime/releases/tag/v0.7.1) pointing to this commit:

```suggestion
  set(UNIFIED_RUNTIME_TAG v0.7.1)
```